packages/arb-token-bridge-ui/src/token-bridge-sdk/hooks/useArbTokenBridge.ts

```diff
@@ -35,13 +37,19 @@ import {
   NodeBlockDeadlineStatusTypes
 } from './arbTokenBridge.types'
 import { useBalance } from './useBalance'
-import { getUniqueIdOrHashFromEvent } from '../util/migration'
-import { getL1TokenData, isClassicL2ToL1TransactionEvent } from '../util'
+import { getL1TokenData } from '../../util/TokenUtils'
+import { getL1ERC20Address } from '../../hooks/useERC20L1Address'
 
 export const wait = (ms = 0) => {
   return new Promise(res => setTimeout(res, ms))
 }
 
+function isClassicL2ToL1TransactionEvent(
+  event: L2ToL1TransactionEvent
+): event is EventArgs<ClassicL2ToL1TransactionEvent> {
+  return typeof (event as any).batchNumber !== 'undefined'
```

Comment:
```suggestion
  return typeof (event as EventArgs<ClassicL2ToL1TransactionEvent>).batchNumber !== 'undefined'
```

Should we do that to avoid using any here?